README.md

```diff
@@ -604,6 +605,35 @@ This library partially supports [inline schema definition dereferencing]( http:/
 
 Note that it only supports local definition referencing, we do not plan on fetching foreign schemas over HTTP anytime soon. Basically, you can only reference a definition from the very schema object defining it.
 
+## Troubleshooting
+
+### Build error: Cannot find module 'buffertools' from  .../node_modules/react-jsonschema-form/node_modules/deeper
```

Comment:
*Build error wrt missing "buffertools" module*

This is mainly to avoid failures/erroneous anchors with the automatic TOC generation, which you should run using the `npm run build:readme` command.